src/util/TimeUtil.js

```diff
@@ -0,0 +1,93 @@
+define(['parse-ms', '@sindresorhus/to-milliseconds'], function (parseMs, toMilliseconds) {
+
+  const MOMENT_UNIT = {
+    MILLISECOND: 'milliseconds',
```

Comment:
Do we need to distinguish singulars and plurals? (i.e.: '1 millisecond' vs. ' 2 milliseconds')

Reply:
good question!
I wouldn't worry it for now given this is a refactor work and trying to retain identical functionality to the TimeUtil from Courage to mitigate risk.
certainly will revisit once it becomes a problem.